ui/helpers/utils/error-utils.test.js

```diff
@@ -0,0 +1,47 @@
+import { SUPPORT_LINK } from '../constants/common';
+import { getErrorHtml } from './error-utils';
+
+const defaultState = {
```

Comment:
What do you think about putting these strings in the test itself? It seems like it'd be more readable. There's also not really a concept of "state" here so the variable seems misleading.

Reply:
Yeah, I noticed that `jest.mock` has to be called before the test is initialised or I get a weird error that locale can't be initialised because only absolute path is supported (calling jest.mock inside a beforeAll or directly inside describe give this error). 

I renamed the variable.